File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
  return s
    .split(" ")
    .map((cur) => {
      if (cur == "") {
        return cur;
      }
      cur = cur.toLowerCase().split("");
      console.log(cur);
      cur[0] = cur[0].toUpperCase();
      return cur.join("");
    })
    .join(" ");
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s = list(s)
    for i in range(len(s)):
        if i == 0:
            s[i] = s[i].upper()
        elif s[i-1] == " ":
            s[i] = s[i].upper()
        else:
            s[i] = s[i].lower()
    return "".join(s)